app/components/conversation-pane/component.js

```diff
@@ -87,7 +87,7 @@ export default Component.extend({
                 this.get('conversation').clearUnreadState();
             }
 
-            if ((key === 'Enter' || keyCode === 13) && !shiftKey) {
+            if ((key === 'Enter' || keyCode === 13) && !shiftKey && keyCode !== 229) {
```

Comment:
what does key code 229 mean? maybe we should put both it and 13 in constants with names just to be a little less vague, my bad for not doing that initially with 13

Reply:
From what I can tell, 229 means a special enter of some kind, generally when using Mac's ability to type in other languages' alphabets.

Reply:
Also I'll go ahead and add those consts :D